### PR TITLE
Clarify Mailserver spec

### PR DESCRIPTION
**Pull request by adambabik**

### CODE REVIEW

File: status-whisper-mailserver-spec.md [hunk 1]
```diff
@@ -3,63 +3,54 @@
...
-
-Since mailservers rely on being online to receive messages on behalf of other clients, this puts a high-availability requirement on individual nodes.
-
-In practice, it is best to treat individual nodes as a form of a cache, and ensure consistency of messages at a different layer. See data sync layer.
+All Whisper envelopes are encrypted. Mailserver node can not inspect their contents.
 
 ### Altruistic and centralized operator risk
 
-In order to be useful, a mailserver has to be online most of time. That means
+In order to be useful, a mailserver SHOULD be online most of time. That means
```
Comment (oskarth) [inline status-whisper-mailserver-spec.md:53]: Perhaps write what happens if they are not

File: status-whisper-mailserver-spec.md [hunk 2]
```diff
@@ -3,63 +3,54 @@
...
-TBD.
-
-## Introduction
+## Abstract
 
-In the case of mobile clients which are often offline, there is a strong need to have an ability to download offline messages. By offline messages, we mean messages sent into the Whisper network and expired before being collected by the recipient. A message stays in the Whisper network for a duration specified as `TTL` (time-to-live) property.
+Being mostly offline is an intrinsic property of mobile clients. They need to save network transfer and battery consumption to avoid spending too much money or constant charging. Whisper protocol, on the other hand, is an online protocol. Messages are available in the Whisper network only for short period of time calculate in seconds.
 
-See [EIP-627](https://eips.ethereum.org/EIPS/eip-627) for more detail on *Whisper Mail Server* and *Whisper Mail Client*.
+Whisper Mailserver is a Whisper extension that allows to store messages permamently and deliver them to the clients even though they are already not available in the network and expired.
```
Comment (oskarth) [inline status-whisper-mailserver-spec.md]: permamently typo

File: status-whisper-mailserver-spec.md (hunk 1 above)
Comment (adambabik) [reply status-whisper-mailserver-spec.md:53]: I think it's obvious after reading Abstract :)

Comment (oskarth) [reply status-whisper-mailserver-spec.md:53]: ok

**Review by oskarth — APPROVED** [review]

**Pull request merged**